### PR TITLE
Migrate off of the Docker Hub images into GitHub packages

### PR DESCRIPTION
**Pull request by perk-sumo**
###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Comment by perk-sumo** [comment]
@astencel-sumo the scheme is different for every chart.
Which particular one you mean?

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -1588,7 +1591,7 @@ otelcol:
     ## Memory Ballast size should be max 1/3 to 1/2 of memory.
     memBallastSizeMib: "683"
     image:
-      name: "sumologic/opentelemetry-collector"
+      name: "ghcr.io/sumologic/opentelemetry-collector"
```
Comment (pmm-sumo) [inline deploy/helm/sumologic/values.yaml:1594]: Is this automatically published somehow?

**Comment by perk-sumo** [comment]
As for the `image.tag` - it's there: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1168/files#diff-da997ab241c9344f2bb085822fe04e4e43eaf0f8034a9d15dc3942b8a5fbe046R6

File: deploy/helm/sumologic/values.yaml [hunk 2]
```diff
@@ -1560,6 +1560,9 @@ prometheus-operator:
 falco:
   ## Set the enabled flag to true to enable falco.
   enabled: false
+  image:
+    registry: ghcr.io
+    repository: sumologic/falco
```
Comment (astencel-sumo) [inline deploy/helm/sumologic/values.yaml:1565]: Isn't `image.tag` missing?

File: deploy/helm/sumologic/values.yaml (hunk 1 above)
Comment (perk-sumo) [reply deploy/helm/sumologic/values.yaml:1594]: Nope, I moved the images by hand for now. We'd need to automate that for future use.

File: deploy/helm/sumologic/values.yaml (hunk 2 above)
Comment (perk-sumo) [reply deploy/helm/sumologic/values.yaml:1565]: The `tag` is used separately, see https://github.com/falcosecurity/charts/blob/master/falco/values.yaml#L6

**Review by pmalek-sumo — APPROVED** [review]
@perk Can we please create a task for automating this?

File: deploy/helm/sumologic/values.yaml (hunk 2 above)
Comment (astencel-sumo) [reply deploy/helm/sumologic/values.yaml:1565]: Right, no need to override `tag` because we do not modify it.

**Review by sumo-drosiek — APPROVED** [review]

File: deploy/helm/sumologic/values.yaml (hunk 1 above)
Comment (perk-sumo) [reply deploy/helm/sumologic/values.yaml:1594]: Task for automating this: https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1175

**Comment by perk-sumo** [comment]
@pmalek-sumo of course! https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1175

**Comment by perk-sumo** [comment]
PR for the v2.1 can be found here: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1442

**Pull request merged**